target/linux/octeon/image/Makefile

```diff
@@ -47,4 +47,12 @@ define Device/ubnt_edgerouter-lite
 endef
 TARGET_DEVICES += ubnt_edgerouter-lite
 
+USG3P_CMDLINE:=-mtdparts=phys_mapped_flash:512k(boot0)ro,512k(boot1)ro,64k(eeprom)ro root=/dev/sda2 rootfstype=squashfs,ext4 rootwait
+define Device/usg3p
```

Comment:
we recently switched to use full names on ubiquiti devices, so this should be
Device/ubnt_unifi-security-gateway-3p
(what's the 3p exactly, i didn't find it during quick search?)

Reply:
As far as I understand, it's the smaller, 3-port edition of the USG. In the Unifi controller the device shows up as "Unifi Security Gateway 3P".